components/builder-worker/src/runner/studio.rs

```diff
@@ -80,6 +88,10 @@ impl<'a> Studio<'a> {
 
         let mut cmd = self.studio_command()?;
         cmd.current_dir(self.workspace.src());
+        if dev_mode && cfg!(not(windows)) {
+            cmd.env("HOME", "/hab/svc/builder-worker/data");
+        }
```

Comment:
It would be useful for future maintainers if some of the discussion you wrote up in the PR message was included at this point as a comment. While someone could figure out why this was written like it is, it would require some Git archaeology; it's also not necessarily obvious from simply reading the code.

Reply:
Addressed in f1935f1